wordpress-mobile/WordPress-iOS
# Prologue Carousel: Extend the top container below button blur view

Project Issue: #15056 
Closes: #15073 
Auth PR:  https://github.com/wordpress-mobile/WordPressAuthenticator-iOS/pull/500

This PR adjust the top container to extend below the button blur view for the unified prologue carousel. Note that the original prologue carousel should not change position and should not have any noticeable changes.

**Not in this PR**
- the correct positioning for the headline text in the unified prologue carousel

### To test
**Test 1**, unified prologue carousel

1. Log out if logged in.
2. Notice the blurred blue background.

**Test 2**, original prologue carousel

1. Set the feature flag `unifiedPrologueCarousel` to `false`
2. Log out if logged in.
3. Notice the carousel. There should not be any changes made to it. (Nothing is positioned oddly).

### Screenshots
| Original | Unified |
| --- | --- |
| ![Simulator Screen Shot - iPhone 11 Pro - 2020-10-12 at 10 59 04](https://user-images.githubusercontent.com/1062444/95772011-adbb9b80-0c81-11eb-9f11-25cde817d7fe.png) | ![Simulator Screen Shot - iPhone 11 Pro - 2020-10-12 at 10 44 59](https://user-images.githubusercontent.com/1062444/95772039-b7450380-0c81-11eb-8cb1-b3b4d353788f.png) |


PR submission checklist:
